tests/fe/fe_values_view_30.cc

```diff
@@ -109,7 +109,7 @@ void test (const Triangulation<dim> &tr,
 
   MappingQGeneric<dim> mapping(1);
 
-  const QGauss<dim> quadrature(2);
+  const QGauss<dim> quadrature(fe.degree+2);
```

Comment:
What was the reason for this change? Did you just want to check more data, or did it actually uncover an issue?